pandas/core/common.py

```diff
@@ -1596,7 +1596,7 @@ def is_timedelta64_dtype(arr_or_dtype):
 
 
 def is_timedelta64_ns_dtype(arr_or_dtype):
-    tipo = _get_dtype_type(arr_or_dtype)
+    tipo = _get_dtype(arr_or_dtype)
```

Comment:
ahh I c, you are right. ok to fix then and pls update the tests in test_common.py